### ISSUE TITLE
win32 releases are mislabeled

### ISSUE THREAD

**Issue by native-api** [github/hub#974]
At https://github.com/github/hub/releases, the win32 release is named "hub-windows-386-2.2.1.tar.gz".
But, it is actually compiled for Pentium 4 (and won't run on processors without SSE2).
So, it should be labelled "[i786](https://myonlineusb.wordpress.com/2011/06/08/what-is-the-difference-between-i386-i486-i586-i686-i786/)" instead.

**Comment by mislav** [comment]
Thanks for reporting. Does you mean you are unable to use hub because you have a processor that doesn't support "SSE2"? I'm sorry about that.

Unfortunately, Go's cross-compilation abilities only support Windows "386" and "amd64" modes. I don't know what would be involved in making a true 386 binary (sans actually compiling it on such machine).

**Comment by native-api** [comment]
Exactly, it throws an "illegal instruction" exception early on on my
machine.

I'm not asking you to make a "true i386 release". I'm probably a vast
minority and as such, can be expected to compile the program myself if
I really need it. I'm just asking you to avoid confusing people.

If you are actually considering this, judging by
https://golang.org/doc/install/source, this appears to be governed by
the $GO386 option.
